easybuild/easyconfigs/p/PhyloBayes-MPI/PhyloBayes-MPI-20161021-intel-2016b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'MakeCp'
+
+name = 'PhyloBayes-MPI'
+version = '20161021'
+commit = 'e4ea1ca'
+
+homepage = 'https://github.com/bayesiancook/pbmpi'
+description = "A Bayesian software for phylogenetic reconstruction using mixture models"
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = ['https://github.com/bayesiancook/pbmpi/archive/']
+sources = ['%s.tar.gz' % commit]
+
+start_dir = 'sources'
+
+# get rid of included binaries to ensur they're recompiled
+prebuildopts = "rm -f data/* &&"
+
+buildopts = 'CC="$MPICXX" CPPFLAGS="$CXXFLAGS -c" LDFLAGS="$CXXFLAGS"'
```

Comment:
because the `Makefile` hardcodes `-O3`, there aren't any flags in there to link with libraries or anything:

```
$ head -3 sources/Makefile 
CC=mpic++
CPPFLAGS= -w -O3 -c
LDFLAGS= -O3
```